src/services/mongoose/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { Types } from "mongoose"
+
+export const toObjectId = <T extends string>(id: T): Types.ObjectId => {
```

Comment:
there is not much benefits to do `<T extends string>` here IMO. would be better to do `export const toObjectId = (id: AccountId | UserId):`?